swagger/v1/swagger.json

```diff
@@ -0,0 +1,483 @@
+            "schema": {
+              "type": "string"
+            },
+            "example": "javascript, css"
+          },
+          {
+            "name": "tags_exclude",
+            "in": "query",
+            "required": false,
+            "description": "Using this parameter will retrieve articles that do _not_ contain _any_\nof comma-separated tags. Articles will be ordered by descending popularity.",
+            "schema": {
+              "type": "string"
+            },
+            "example": "node, java"
+          },
+          {
+            "name": "username",
+            "in": "query",
+            "required": false,
+            "description": "Using this parameter will retrieve articles belonging\n            to a User or Organization ordered by descending publication date.\n            If `state=all` the number of items returned will be `1000` instead of the default `30`.\n            This parameter can be used in conjuction with `state`.",
+            "schema": {
+              "type": "string"
+            },
+            "example": "ben"
+          },
+          {
+            "name": "state",
+            "in": "query",
+            "required": false,
+            "description": "Using this parameter will allow the client to check which articles are fresh or rising.\n            If `state=fresh` the server will return fresh articles.\n            If `state=rising` the server will return rising articles.\n            This param can be used in conjuction with `username`, only if set to `all`.",
+            "schema": {
+              "type": "string",
+              "enum": ["fresh", "rising", "all"]
+            },
+            "example": "fresh"
+          },
+          {
+            "name": "top",
+            "in": "query",
+            "required": false,
+            "description": "Using this parameter will allow the client to return the most popular articles\nin the last `N` days.\n`top` indicates the number of days since publication of the articles returned.\nThis param can be used in conjuction with `tag`.",
+            "schema": {
+              "type": "integer",
+              "format": "int32",
+              "minimum": 1
+            },
+            "example": 2
+          },
+          {
+            "name": "collection_id",
+            "in": "query",
+            "required": false,
+            "description": "Adding this will allow the client to return the list of articles\nbelonging to the requested collection, ordered by ascending publication date.",
+            "schema": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "example": 99
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A List of Articles",
+            "content": {
+              "application/json": {
+                "example": [
```

Comment:
All of the example data in this section is generated our rspec factories during Rspec test execution!